.github/scripts/cleanup-cluster.sh

```diff
@@ -52,5 +52,14 @@ for ns in $namespaces; do
         break
     fi
     echo "deleting namespaces: $ns"
+
+    # Remove finalizers from the namespace if present
+    kubectl get namespace $ns -o json | jq '.spec.finalizers = []' >temp-namespace.json
+    kubectl replace --raw "/api/v1/namespaces/$ns/finalize" -f ./temp-namespace.json
```

Comment:
Does `--force` help here? https://kubernetes.io/docs/reference/kubectl/generated/kubectl_delete/

Just thinking it might be easier if it works.

I'm also curious what the finalizers are. We have to be a little bit careful with this, because certain types in Kubernetes will manage cloud resources. If we delete the Kubernetes resource without running the finalizers, then the cloud resources leak.